parallax/mock_sim.py

```diff
@@ -0,0 +1,375 @@
+import pyqtgraph as pg
+import cv2 as cv
+import numpy as np
+import coorx
+from .calibration import CameraTransform, StereoCameraTransform
+from .lib import find_checker_corners
+from .config import config
+from .threadrun import runInGuiThread
+
+
+class CameraTransform(coorx.CompositeTransform):
```

Comment:
feels like a duplicate of same class name in calibration.py, think about this (chris)